src/schedules/dto/get-topics.response.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Topic } from '../domain/topic';
+import { PaginationMeta } from 'src/common/dto/query-pagination.dto';
+
+export class GetTopicsResponse {
+  @ApiProperty({ type: [Topic] })
+  data: Topic[];
+
+  @ApiProperty({ type: PaginationMeta })
+  meta: PaginationMeta;
+}
```

Comment:
Não precisa importar o PaginationMeta especificamente, mas sim o PaginationResponse assim:

```suggestion
export class GetTopicsResponse extends PaginationResponse<Topic> {}
```